packages/augur-state/src/index.ts

```diff
@@ -1,27 +1,21 @@
-export {API} from "./api/API";
-export {ServerController} from "./server/ServerController";
-export {Controller} from "./Controller";
+import RunWorker from "./Sync.worker";
 
-import RunWorker from "./run.worker";
+// start the web worker and move console.log over to paint in the browser
+console.log = (msg: string) => {
```

Comment:
Does this mess with `console.log` calls made elsewhere?

Reply:
I left that there thinking it would be removed when augur-ui gets hooked up. However, you make a good point and I should just remove it.